airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableRow.module.scss

```
@@ -38,11 +47,35 @@
   &.selected {
     background-color: colors.$blue-transparent;
   }
+
+  &.disabled {
+    background-color: colors.$grey-50;
+  }
 }
 
 .streamRowCheckboxCell {
+  margin-right: variables.$spacing-sm;
+  max-width: 43px;
+  text-align: center;
+  font-size: 10px;
+  line-height: 13px;
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
-  padding-left: 27px;
+  padding-left: calc(#{variables.$spacing-xl} + #{variables.$spacing-sm});
```

These numbers can be added with sass so that the result is a hard value in CSS:

```suggestion
  padding-left: variables.$spacing-xl + variables.$spacing-sm;
```